src/pages/Attendance.jsx

```diff
@@ -1,11 +1,220 @@
+import React from "react";
+import dayjs from "dayjs";
+import "dayjs/locale/ko";
+import { Stack, styled } from "@mui/material";
+import Chip from "@mui/material/Chip";
+import Typography from "@mui/material/Typography";
+import Checkbox from "@mui/material/Checkbox";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { MobileDatePicker } from "@mui/x-date-pickers/MobileDatePicker";
+import CalendarMonthIcon from "@mui/icons-material/CalendarMonth";
+import ChurchIcon from "@mui/icons-material/Church";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+import SettingsIcon from "@mui/icons-material/Settings";
 import Layout from "../components/Layout/Layout";
```

Comment:
```suggestion
import SettingsIcon from "@mui/icons-material/Settings";

import Layout from "../components/Layout/Layout";
```
외부라이브러리/ 내부 import 사이에는 개행 구분